proposals/819-kubeflow-sdk/README.md

```diff
@@ -0,0 +1,423 @@
+# KEP-819: Kubeflow SDK for ML Experience
+
+## Summary
+
+Original Google document: https://docs.google.com/document/d/1rX7ELAHRb_lvh0Y7BK1HBYAbA0zi9enB0F_358ZC58w/edit
+
+This KEP outlines a proposal to design Kubeflow SDK that streamlines user experience for
+data scientists and ML engineers to interact with various Kubeflow projects
+(Trainer, Katib, Spark Operator, Pipelines, Model Registry, KServe).
```

Comment:
I see Feast is left out and my guess is that's because of the outstanding question on whether it will be donated to Kubeflow, which is reasonable but still leaves a rather large gap in the product experience.

Reply:
That's fine, that said it doesn't feel like we'll make a quick decision on Feast given the desire to have user interviews/testimonials and I personally think that's for the worse of the community. 

I won't block progress, I'll just be transparent about my opinion.